app/policies/post_policy.rb

```diff
@@ -4,7 +4,7 @@ class PostPolicy < ApplicationPolicy
   def create?
     return false unless group&.has_post_type?(post_type.id)
     return true if leader?
-    return true if group == Group.sssl && evaluation_helper? &&
+    return true if evaluation_helper? &&
```

Comment:
I thought about that a lot when I had created the policies. In my opinion this fomat is batter for complex conditions, that can change over time, because they are easier to modify, and can produce a cleaner git history. Also in some place it can be more understandable if we use this format.

Reply:
I agree with @SepsiLaszlo, it's more clear if we use the same pattern in every condition.